movies-app/src/components/SortControl/SortControl.jsx

```diff
@@ -1,34 +1,31 @@
-import React, { useState } from "react";
+import React from "react";
 import "./SortControl.css";
 import { useSearchParams, useLocation } from "react-router-dom";
 
 const SortControl = () => {
+
   const { search } = useLocation();
   const [searchParams, setSearchParams] = useSearchParams({});
```

Comment:
Nem kell neki az üres object, buogokat tud okozni,mert így felül írhatja egy üres objecttel ha van valami az urlben
```suggestion
  const [searchParams, setSearchParams] = useSearchParams();
```